src/Illuminate/Foundation/Vite.php

```diff
@@ -108,7 +220,80 @@ public function reactRefresh()
     }
 
     /**
-     * Generate an appropriate tag for the given URL.
+     * Make tag for the given chunk.
+     *
+     * @param  string  $url
+     * @param  array  $chunk
+     * @param  array  $manifest
+     * @return string
+     */
+    protected function makeTagForChunk($url, $chunk, $manifest)
+    {
+        if (
+            static::$nonce === null
+            && ! array_key_exists(static::$integrityKey, $chunk)
+            && static::$scriptTagAttributesResolvers === []
+            && static::$styleTagAttributesResolvers === []) {
+            return $this->makeTag($url);
+        }
```

Comment:
This is in place so that the introduction of these features is not a breaking change.

We can remove this and tidy things up in `10.x`.